src/azure-cli/HISTORY.rst

```diff
@@ -3,9 +3,11 @@
 Release History
 ===============
 
-**AMS**
+**Ams**
 
-* BREAKING CHANGE: Changed job creation from "job start" to "job create".
+* [BREAKING CHANGE]
```

Comment:
Can you re-edit your release note reference breaking change in HD insught?

Reply:
Edited.

Thanks
Jinlin